README.md

```diff
@@ -498,6 +506,54 @@ last_modified: Date;
 - GEOMETRY
 - JSON
 
+## Extended operators
+MySQL connector supports the following MySQL-specific operators:
+- [`match`](#operator-match)
+Please note extended operators are disabled by default, you must enable
+them at datasource level or model level by setting `allowExtendedOperators` to
+`true`.
+### Operator `match`
+The `match` operator allows you to perform a full text search using the [MATCH() .. AGAINST()](https://dev.mysql.com/doc/refman/8.0/en/fulltext-search.html) operator in MySQL.
+
+Three different modes of the `MATCH` clause are also available in the form of operators - 
+
+- `boolmatch` for [Boolean Full Text Search](https://dev.mysql.com/doc/refman/8.0/en/fulltext-boolean.html)
```

Comment:
as requested add `nlqematch` or the suggested name `matchnlqe` to support `IN NATURAL LANGUAGE MODE WITH QUERY EXPANSION`

Reply:
From what I understand from [here](https://dev.mysql.com/doc/refman/8.0/en/fulltext-query-expansion.html), `IN NATURAL LANGUAGE MODE WITH QUERY EXPANSION` is equivalent to `WITH QUERY EXPANSION` .

Reply:
> From what I understand from [here](https://dev.mysql.com/doc/refman/8.0/en/fulltext-query-expansion.html), `IN NATURAL LANGUAGE MODE WITH QUERY EXPANSION` is equivalent to `WITH QUERY EXPANSION` .

Yes, but in terms that they both enable `blind query expansion` , however they have it listed a separate modifier [Here](https://dev.mysql.com/doc/refman/8.0/en/fulltext-search.html) , meaning they need to support it as well in case the internal implementation changes from the `mysql` side.

```sql
search_modifier:
  {
       IN NATURAL LANGUAGE MODE
     | IN NATURAL LANGUAGE MODE WITH QUERY EXPANSION
     | IN BOOLEAN MODE
     | WITH QUERY EXPANSION
  }
```

Reply:
Oh right, I have made the required changes for this.